### ISSUE TITLE
Metro 4: Completely off the air

### ISSUE THREAD

**Issue by mgenelins** [mgenelins/w0yc-repeaters#26]
Metro 4 was reported off-air September 19, 2023 to Matt via Zello 'Metro Twin Cities MN' channel.

Matt verified September 19 and September 20th that the common 141.3hz PL tone no longer seems to bring up Metro 4.

There has been some jamming in the last few months as well as someone with a "echo back simplex repeater" on the 141.3 hz pl tone, so checking the input while on-site might be a good first step.

- [x] Site visit to check on the Metro 4 462.650 repeater.
- [x] Site visit and use a HT to check the input of 467.650 for jamming or noise.

**Comment by KD0TLS** [comment]
Don’t mean to pile on the bad news, but Metro 2 is also down; no ID, no kerchunk. If a power cycle doesn’t fix it, I’ll take it in to Action Radio. I can get that short hang time and CW beacon rate fixed at the same time.  Todd D.   From: Matt GenelinSent: Wednesday, September 20, 2023 11:11 AMTo: mgenelins/w0yc-repeatersCc: SubscribedSubject: [mgenelins/w0yc-repeaters] Metro 4: Completely off the air (Issue #26) Metro 4 was reported off-air September 19, 2023 to Matt via Zello 'Metro Twin Cities MN' channel.Matt verified September 19 and September 20th that the common 141.3hz PL tone no longer seems to bring up Metro 4.There has been some jamming in the last few months as well as someone with a "echo back simplex repeater" on the 141.3 hz pl tone, so checking the input while on-site might be a good first step.[ ] Site visit to check on the Metro 4 462.650 repeater.[ ] Site visit and use a HT to check the input of 467.650 for jamming or noise.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by mgenelins** [comment]
@KD0TLS Sorry, I was at the site and moved the antenna port for Metro 2 over directly onto the repeater. It shares an antenna with the APRS unit I removed from service today, and I just remembered that I need to turn Metro 2's power back on.

I created a new issue for this:
https://github.com/mgenelins/w0yc-repeaters/issues/29

I'll see if Ian is still on campus and can run up for this "press the power button" job.
